README.md

```diff
@@ -38,12 +39,13 @@ SSH key based authentication configured.
   hosts: localhost
 
   roles:
-    - role: ryankwilliams.ssh_copy_id
+    - role: rywillia.ssh-copy-id
```

Comment:
Could this line be reverted please? I recently renamed my GitHub account and the role on galaxy that will include the latest changes is `ryankwilliams.ssh_copy_id`.

Reply:
Sure, reverted